x-pack/functionbeat/docs/configuring-howto.asciidoc

```diff
@@ -41,8 +41,6 @@ include::{libbeat-dir}/docs/queueconfig.asciidoc[]
 :allplatforms!:
 
 [role="xpack"]
```

Comment:
Shouldn't this be removed as well? I believe it will be appended to [this page](https://www.elastic.co/guide/en/beats/functionbeat/master/configuring-output.html) if not.

Reply:
yup, I want it to be added to  outputconfig.asciidoc. All the pages need to be flagged as x-pack, even the included ones.